components/EpisodeDetails/index.js

```diff
@@ -14,20 +15,51 @@ const StyledArticle = styled.article`
   border-radius: 10px;
 `;
 
+const StyledDetailsTitle = styled.h2`
+  text-align: right;
+  font-size: 2rem;
+`;
+
+const StyledReturnButton = styled.button`
+  position: absolute;
+  top: 0.5rem;
+  left: 0.5rem;
+  border: none;
+  background: none;
+  width: fit-content;
+  height: fit-content;
+`;
+
 const StyledImage = styled(Image)`
-  width: 90%;
+  width: 100%;
   height: auto;
   align-self: center;
+  margin: 1rem;
+`;
+
+const StyledFoldButton = styled.button`
+  font-size: 1.2rem;
+  margin-top: 0.8rem;
+`;
+
+const StyledNoContentMessage = styled.p`
+  margin-top: 0.8rem;
+  color: lightgrey;
 `;
 
 export default function EpisodeDetails({ episode }) {
+  const [showDescription, setShowDescription] = useState(false);
+  const [showTracklist, setShowTracklist] = useState(false);
+
   const router = useRouter();
   const {
     nummer,
     titel,
     autor,
     hörspielskriptautor,
+    beschreibung,
     veröffentlichungsdatum,
+    kapitel,
     links,
   } = episode;
```

Comment:
I know it's a german application.
Usually we should use english for the naming of variables.

Reply:
changed it